presentation-api/controlling-ua/PresentationRequest_onconnectionavailable-manual.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>PresentationRequest.onconnectionavailable (manual test)</title>
+<link rel="author" title="Tomoyuki Shimizu" href="https://github.com/tomoyukilabs">
+<link rel="help" href="https://w3c.github.io/presentation-api/#starting-a-presentation">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<p>Click the button below and select the available presentation display, to start the manual test.</p>
+<button id="presentBtn" onclick="startPresentation()">Start Presentation Test</button>
+
+
+<script>
+    // ----------
```

Comment:
Since this is a manual test, there needs to be a call somewhere to:
`setup({explicit_timeout: true});`

The test harness will time out after a few seconds otherwise. Note that one could argue that the test harness should automatically disable the timeout for tests that have "-manual" in their name. That's note the case today, though, so we need to do it explicitly in the code.

(I note I missed that in previous tests too... We'll fix other tests later on, no big deal)